src/logging.h

```diff
@@ -21,7 +21,7 @@
 #define PROTOLOGFILE "web100srvprotocol.log"  // name of protocol log file
 #define PROTOLOGPREFIX "web100srvprotocol_"  // prefix for protocol log file
 #define PROTOLOGSUFFIX ".log"  // suffix for protocol validation log file
-#define FILENAME_SIZE 256
+#define FILENAME_SIZE 1024
```

Comment:
Future TODO.

While trying to decide if this value was sufficient or not, I looked under a rock in writeMeta(). Please include the FILENAME_SIZE and the various constants used in writeMeta as part of "TODO: ensure every write to a char array will always result in a null-terminated string of less than the buffer size."

There are several assumptions in writeMeta that don't add up. I think this value (based on how its used) wants to be PATH_MAX from <linux/limits.h>, as do some of the static buffer sizes in writeMeta(). There is a convention of using the client DNS name in the file name of logs when reverse lookup is successful. But there is a mismatch between the maximum DNS name and NAME_MAX from <linux/limits.h> and the other filename elements added by ndt.